src/rest_server.cpp

```diff
@@ -85,10 +85,14 @@ void rest_server::doClassificationPost(const Rest::Request &request,
   response.headers().add<Http::Header::AccessControlAllowOrigin>("*");
   nlohmann::json j = nlohmann::json::parse(request.body());
   int count = 10;
+  float threshold = 0.0;
```

Comment:
Could be a member of the class `classifier`?

Reply:
The threshold can change at each classification called, it doesn't seem to make sense to make it a `classifier` class member.